english/coffee/another-coffee/another-coffee.md

```diff
@@ -4,4 +4,6 @@ You are quite hungry aren't you ?
 
 [Eat a nice pizza slice](../chilli/pizza-with-chilli.md)
 
-[Eat a bagel](../bagel/eat-a-bagel.md)
\ No newline at end of file
+[Eat a bagel](../bagel/eat-a-bagel.md)
+
+[Have another coffee?](../another-coffee.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.